sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -118,6 +119,119 @@ public void valid() throws Exception {
 
     List<String> debugLogs = logTester.logs(LoggerLevel.DEBUG);
     assertThat(debugLogs.get(0)).startsWith("The current user dir is '");
+
+    // FIXME the test case may be wrong
```

Comment:
Should we add an issue or card for this to be sure we don't forget to address it?